json-spec/extensions/stac-eo-spec.md

```diff
@@ -0,0 +1,60 @@
+# STAC EO Extension JSON Spec
+
+This document explains the fields of the STAC Earth Observation (EO) Extension to a STAC `Item`. EO data is considered to be data that represents a snapshot of the earth for a single date and time. It could consist of multiple spectral bands in any part of the electromagnetic spectrum. Examples of EO data include sensors with visible, short-wave and mid-wave IR bands (e.g., the OLI instrument on Landsat-8), long-wave IR bands (e.g. TIRS aboard Landsat-8), as well as SAR instruments (e.g. Sentinel-1).
+
+## EO Extension Description
+These are the additional fields to appear under a STAC `Item`, as well as the items's assets field and `Collection` level metadata.
+
+### `Item` additions
+| element             | type info                 | name                    | description                                                                                 | 
+|----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
+| eo:collection             | string                      | Collection Name       | The name of the STAC `Collection` this `Item` belongs to |
+| eo:crs     | reference system    | ref system             | CRS of the datasource in full WKT format. null if no crs
+| eo:cloud_cover     | number (optional)   | Cloud Cover Pct    | Percent of cloud cover | 
+| eo:off_nadir      | number (optional)   | Off nadir    | The viewing angle measured from nadir, in degrees
+| eo:azimuth      | number (optional)   | Azimuth    | The viewing azimuth angle, in degrees
+| eo:sun_azimuth    | number (optional)   | Sun Azimuth | The sun azimuth angle, in degrees
+| eo:sun_elevation  | number (optional)   | Sun Elevation | The sun elevation angle, in degrees
+| eo:supported_assets | array of asset names | Supported Assets | A list of the key names for the asset types that are available now.
```

Comment:
Oh, I was just saying we should make that line a bit more clear.

Hrm, but I am having trouble remembering the exact utility.

Reply:
Maybe if for instance the asset in on a 'Requester Pays' bucket? In that case we could have a s3://... reference but the file would not be 'downloadable' in the sense that it is typically used - that is, no http:// or ftp:// reference.

Reply:
I agree with Matt's comment that the eo:supported_assets is a repeat of the contents of the assets.  I don't think we need this, unless people think this will be something people commonly search on.

Reply:
Yeah, I'm inclined to leave it off for right now. Perhaps we put it in an 'ideas' / 'future directions' section, and see if / when people want to use it. But since the utility isn't super clear at the moment I'm inclined to not put it in the spec for right now. But @fredliporace - definitely would be great for you to play with a way to specify requestor pays type stuff.

Reply:
@alkamin or @kasey   do you remember what the reason was for a supported_assets field?

Reply:
The intent was for searching as we expect some people to know what type of assets they need for the analysis they are doing.

Reply:
I'm leaving it off for now. We can add it back in if we find it's of use